docs/_static/js/banner.js

```diff
@@ -14,23 +14,24 @@ function hasCookie(name, value) {
+      <div>
+        <strong>Wagtail Space US</strong><br> 24 – 26 March 2022 online and in Cleveland, Ohio
+      </div>
+    </a>
+    <button class="wagtailspace-close" aria-label="Hide banner" aria-expanded="true">×</span>
+  </div>
+  `);
 
   if (
     !hasCookie('wagtailspaceClosed', 'true') &&
-    new Date() < new Date(2019, 2, 15)
+    new Date() < new Date(2022, 2, 16)
```

Comment:
Should be the 26th I think...?
```suggestion
    new Date() < new Date(2022, 2, 26)
```